src/cubical/dune

```diff
@@ -4,5 +4,5 @@
   (:standard -w -32-38 -warn-error -a+31))
  (preprocess
   (pps ppx_deriving.std))
- (libraries ezjsonm uuseg uuseg.string uutf cooltt.basis)
```

Comment:
I'm surprised that the unicode libraries aren't used... don't we print anything out in `cubical`?

Reply:
There are lots of printers under `cubical/`, but none of them seems to be using proper Unicode? (Not that I like it. In fact, this could be seen as a defect. :laughing:)

Reply:
oh ,ok. Well I don't really care haha